### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by mustafakamran98** [capjamesg/openai-blog-rss#2]
I don't know if you still have an interest in this project, but I'd figure it's worth mentioning: OpenAI recently revamped their website, and their official RSS feed no longer works, nor can I seem to find the link to the new one if it exists. This also may have affected this project as I see nothing from your feed.

**Comment by capjamesg** [comment]
Oh, this will have an effect for sure. I'll take a look over the next day or two and get this working again with the new site. Thank you for raising this! I'll post an update here when everything is back online.

**Comment by capjamesg** [comment]
I have pushed an update to this codebase with revised, working code. I will deploy this tomorrow to https://jamesg.blog/openai.xml.

**Comment by felixbuenemann** [comment]
@capjamesg The feed is currently empty.

**Comment by capjamesg** [comment]
OpenAI's website now uses a Cloudflare feature that is preventing against programmatically parsing their pages. I have one or two more things left to try. Getting this back up and running is still very much on my TODO list. Contributions, too, are appreciated!

**Comment by felixbuenemann** [comment]
Yeah, I tried to build a CloudFlare Worker to generate the RSS Feed, but always got a 403 Error when trying to fetch the metadata from the JSON API mentioned in https://github.com/capjamesg/openai-blog-rss/issues/3#issuecomment-2116392720.

**Comment by tofarfar** [comment]
@felixbuenemann @capjamesg  
If you use Node.js, you can bypass it by using the [got-scraping](https://github.com/apify/got-scraping) package.

**Comment by mustafakamran98** [comment]
It seems that OpenAI has brought back their RSS feed again. The old link now redirects to https://openai.com/news/rss.xml. Feel free to close if necessary.
